### DISCUSSION TITLE
How to apply new CSS-in-JS ?

### DISCUSSION THREAD

**Discussion by azro352** [ant-design/ant-design#38879] [Q&A] [answered]
Hi
I'm using [`@ant-design/charts`][1] which has a dependecy to [`x6-react-component`][2] through @ant-design/flowcharts

Here is there actual code, who's failing since antd@v5 , I'd like to make a PR in their project, but I don't know what the fix is related to that new css-in-js 

- issue https://github.com/antvis/X6/issues/2907 
- file https://github.com/antvis/X6/blob/master/packages/x6-react-components/src/toolbar/item.tsx#L5

Could you explain me the change needed so I can make the PR, and we'll be able to use both the charts and hantd@5 ?

Thanks

[1]: https://github.com/ant-design/ant-design-charts
[2]: https://github.com/antvis/X6/tree/master/packages/x6-react-components

**Comment by MadCcc** [comment, marked as answer]
Just remove css import, CSS-in-JS will be imported by default.

**Comment by Mai-Tuyen** [comment]
why css in css-in-js not same with old ~antd/dist/antd.css. my UI is change st. How can i revert UI as ~antd/dist/antd.css in V5. Many thanks for any help !!!
